autoload/lsp_settings/utils.vim

```diff
@@ -68,3 +68,16 @@ function! lsp_settings#utils#load_schemas(name) abort
   let l:schemas = json_decode(join(readfile(s:catalog_path), "\n"))['schemas']
   return extend(l:schemas, lsp_settings#get(a:name, 'schemas', []))
 endfunction
+
+function! lsp_settings#utils#handle_work_done_progress(progress) abort
+  let l:value = a:progress['value']
+  if l:value['kind'] ==# 'end'
+    let g:lsp_progress['messages'] = ''
+    let g:lsp_progress['percentage'] = 100
+  elseif l:value['kind'] ==# 'begin'
+    let g:lsp_progress['title'] = l:value['title']
+  elseif l:value['kind'] ==# 'report'
+    let g:lsp_progress['messages'] = l:value['message']
```

Comment:
`message` and `percentage` is optional.

https://microsoft.github.io/language-server-protocol/specifications/specification-current/#workDoneProgressReport

We might want to do someething like this instead.

```
let g:lsp_progress['messages'] = get(l:value, 'message', '')
let g:lsp_progress['percentage'] = get(l:value, 'percentage', '')
```

Reply:
Thank you.
I fixed it. And I also update this code here. 
https://github.com/micchy326/vim-airline